aarch64/src/l.S

```diff
@@ -0,0 +1,29 @@
+// Aarch64 entry (Raspberry Pi 3, 4 focussed)
+ 
+.section .boottext, "awx"
+.globl start
+
+start:
+	// All cores other than 0 should just hang
+    mrs     x1, mpidr_el1
+    and     x1, x1, #3
+    cbnz    x1, 5f
+	
+	// On the main core, so set up the initial stack
+	ldr	x5, =start
+	add	x5, x5, #4096
+	mov	sp, x5
+
+	// Clean the BSS section
```

Comment:
Does the loader (u-root or whatever) already do this?

Reply:
I haven't seen any reference to the raspberry pi bootloader clearing bss.

Reply:
If it does, we can remove this, but the examples I've seen all clear BSS in one way or another.  (they may all be wrong of course 🤷 )